### ISSUE TITLE
Log non enumerable properties of unknow error objects

### ISSUE THREAD

**Issue by mems** [Adobe-Marketing-Cloud/reactor-uploader#53]
### Expected Behaviour

When the server return an error when uploading a package, an error message is ouput with the error details:

```
Error: Error uploading extension package. An unknown error occurred: {"message": "Newer version already uploaded"}.
```

### Actual Behaviour

The error message is not present:

```
Error: Error uploading extension package. An unknown error occurred: {}.
```

This is due to how properties are defined for errors types: [javascript - Is it not possible to stringify an Error using JSON.stringify? - Stack Overflow](https://stackoverflow.com/questions/18391212/is-it-not-possible-to-stringify-an-error-using-json-stringify)

### Reproduce Scenario (including but not limited to)

Get a server error that will be logged, or a network issue (offline)

#### Steps to Reproduce

1. create an extension 1.0.0
2. upload it
3. release it
4. update the version to 0.9.0
5. upload it

#### Sample Code that illustrates the problem

```js
JSON.stringify(new Error("message", {cause: new Error("message 2")})) === "{}";
```

**Comment by jeffreywalter** [comment]
@adobe export issue to Jira project PDCL as Story

**Comment by brenthosie** [comment]
export issue to Jira project PDCL

**Comment by brenthosie** [comment]
@adobe export issue to Jira project PDCL

**Comment by brenthosie** [comment]
Merged to v5.1.0: https://github.com/adobe/reactor-uploader/pull/54
